eip-5564.md

```diff
@@ -84,7 +84,7 @@ function computeStealthKey(
 
 The implementation of these methods is scheme-specific. The specification of a new stealth address scheme MUST specify the implementation for each of these methods. Additionally, although these function interfaces are specified in Solidity, they do not necessarily ever need to be implemented in Solidity, but any library or SDK conforming to this specification MUST implement these methods with compatible function interfaces.
 
-A one byte integer is used to identify stealth address schemes. A mapping from the scheme ID to it's specification is maintained in **{TODO where to maintain it?}**. An EIP is required to standardize a new stealth address scheme. These EIP extensions MUST specify:
+A one byte integer (`schemeID`) is used to identify stealth address schemes. The `schemeID` represents an incrementing integer beginning at 0. A mapping from the `schemeID` to it's specification MUST be declared in the EIP that proposes to standardize a new stealth address scheme. These EIP extensions MUST specify:
```

Comment:
Nit, I personally prefer `schemeId` over `schemeID` (there's a bunch of places to change this if you're ok with the change)

Reply:
Yeah I agree, I'll change it to schemeId.